src/libsyntax/feature_gate.rs

```diff
@@ -554,6 +554,9 @@ declare_features! (
     // Allows using C-variadics.
     (active, c_variadic, "1.34.0", Some(44930), None),
 
+    // std::{rc,sync}::Weak::{as,from,into}_raw
+    (active, weak_into_raw, "1.36.0", Some(60728), None),
```

Comment:
I don't believe that library-only features need to touch this stuff.

Reply:
They certainly don't.

Reply:
So, the right way is to simply just document it in the unstable book?

Reply:
I've just added them in the library, done. Not sure if we usually document library features in the book.